### PR TITLE
Update hyperbridge adapter to v1.0.0

### PR DESCRIPTION
**Pull request by seunlanlege**
Hyperbridge has been deployed to mainnet. We've updated the SDK to v1.0.0 and removed the hardcoded host addresses.

### CODE REVIEW

File: packages/evm/contracts/adapters/Hyperbridge/HyperbridgeReporter.sol [hunk 1]
```diff
@@ -11,11 +11,18 @@ import { Reporter } from "../Reporter.sol";
 contract HyperbridgeReporter is Reporter, Ownable, BaseIsmpModule {
     string public constant PROVIDER = "hyperbridge";
 
-    constructor(address headerStorage, address yaho) Reporter(headerStorage, yaho) {
-        address host = host();
-        address feeToken = IDispatcher(host).feeToken();
+    // @dev The address of the IsmpHost on the current chain
+    address public _host;
```
Comment (allemanfredi) [inline packages/evm/contracts/adapters/Hyperbridge/HyperbridgeReporter.sol]: same here

File: packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol [hunk 2]
```diff
@@ -10,6 +10,17 @@ contract HyperbridgeAdapter is BlockHashAdapter, Ownable, BaseIsmpModule {
     mapping(bytes32 => bytes32) public enabledReporters;
     mapping(bytes32 => uint256) public chainIds;
 
+    // @dev The address of the IsmpHost on the current chain
+    address public _host;
```
Comment (allemanfredi) [inline packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol]: variable with _` `before the name should be private. If you want to keep it public please remove it.

File: packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol [hunk 3]
```diff
@@ -10,6 +10,17 @@ contract HyperbridgeAdapter is BlockHashAdapter, Ownable, BaseIsmpModule {
...
+
+    constructor(address ismpHost) {
+        _host = ismpHost;
+    }
+
+    function host() public view override returns (address) {
```
Comment (allemanfredi) [inline packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol:20]: why `_host` and `host`? They return the same thing. I'd set `_host` as private

**Review by allemanfredi — CHANGES_REQUESTED** [review]

File: packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol (hunk 2 above)
Comment (seunlanlege) [reply packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol]: Slight oversight

File: packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol [hunk 4]
```diff
@@ -10,6 +10,17 @@ contract HyperbridgeAdapter is BlockHashAdapter, Ownable, BaseIsmpModule {
     mapping(bytes32 => bytes32) public enabledReporters;
     mapping(bytes32 => uint256) public chainIds;
 
+    // @dev The address of the IsmpHost on the current chain
+    address private _host;
```
Comment (allemanfredi) [inline packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol:14]: i would make it `public` and i would remove `function host() public view override returns (address)`

**Review by allemanfredi — CHANGES_REQUESTED** [review]

File: packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol (hunk 4 above)
Comment (seunlanlege) [reply packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol:14]: Unfortunately this method is required by the `BaseIsmpModule`

**Review by allemanfredi — APPROVED** [review]

**Pull request merged**